README.md

```diff
@@ -34,5 +34,7 @@ The main Labels are as follows with the corresponding meaning:
 * [Status: In Progress](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20In%20Progress): A user have mentioned that they have started to work on the request, and/or a new maintainer have been added to the existing package.
 * [Status: Review](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20Review): A package for the current request have been submitted to chocolatey.org, and is currently awaiting a review from the chocolatey community moderators.
 * [Status: Published](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20Published): The package for the current request have been approved on the chocolatey community repository, and is now available for everyone to use.
-* [Blocked Upstream](https://github.com/chocolatey/chocolatey-package-requests/labels/Blocked%20Upstream): There are known issues that need to be resolved before a package can be created for the current request. (This can be everything from needing to embedd a package for a software that does not grant any distribution rights, download is blocked behind a login form, additional headers needed to be sent with the download request, and more...).
-* [Duplicate](https://github.com/chocolatey/chocolatey-package-requests/labels/Duplicate): The current request is a duplicate of a previous open request.
+* [Status: Blocked Upstream](https://github.com/chocolatey/chocolatey-package-requests/labels/Blocked%20Upstream): There are known issues that need to be resolved before a package can be created for the current request. (This can be everything from needing to embedd a package for a software that does not grant any distribution rights, download is blocked behind a login form, additional headers needed to be sent with the download request, and more...).
+* [Status: Duplicate](https://github.com/chocolatey/chocolatey-package-requests/labels/Duplicate): The current request is a duplicate of a previous open request.
+* [Status: High Virus Count](https://github.com/chocolatey/chocolatey-package-requests/labels/High%20Virus%20Count): The installer, or the package once it has been submitted, have been flagged with having a high virus scount, by Virus Total. A high virus count is more than 15% of the scanners that were used by Virus Total on the package / installer flagging an issue.
+* [Status: Not A Package Request / Can't Implement / Invalid](https://github.com/chocolatey/chocolatey-package-requests/labels/Not%20A%20Package%20Request%20%2F%20Can%27t%20Implement%20%2F%20Invalid): The issue raised is not a package or maintainer request, cannot be implemented as a package or is otherwise invalid.
```

Comment:
👍

Reply:
Just a thought, perhaps combining this label and the `Already Exist` label would be worthwhile?
`Already Exist` doesn't really need a label for itself (yeah, I know, I'm the one that added it).

Reply:
Yeah, I agree. That makes sense.